src/components/init-modal/screens/api-keys/index.js

```diff
@@ -109,19 +109,18 @@ export default ( { onSetupStatus } ) => {
 								disabled: true,
 								label: __( 'Select service provider', 'newspack-newsletters' ),
 							},
-							{ value: 'manual', label: __( 'Manual', 'newspack-newsletters' ) },
-							{ value: 'mailchimp', label: __( 'Mailchimp', 'newspack-newsletters' ) },
+							{ value: 'mailchimp', label: 'Mailchimp' },
```

Comment:
Why remove the translation on these strings?

Reply:
Mailchimp is a brand so it shouldn't be translated.